src/MSBuild/Resources/xlf/Strings.cs.xlf

```diff
@@ -1393,13 +1395,13 @@ Copyright (C) Microsoft Corporation. Všechna práva vyhrazena.
     </note>
       </trans-unit>
       <trans-unit id="HelpMessage_32_ProfilerSwitch">
-        <source>  /profileevaluation:&lt;file&gt;    
-                     Profiles MSBuild evaluation and writes the result 
```

Comment:
This whitespace-only change happens during the build when it updates the XLF files.. I suppose I should keep it in bc it needs to be done at some point?

Reply:
It looks like your editor stripped some trailing whitespace when you edited the `.resx` from which these are updated, and the tooling faithfully copied that whitespace difference here. I think that's fine--we shouldn't have the trailing whitespace, and you're generating real diffs for these files anyway.

@cdmihai Do you think we should avoid this? I assume the translation team will see the whitespace change and re-mark these as "translated", but maybe we should do that for them?

Reply:
I'll try to be more careful on the next iteration since I assume there are going to be phrasing changes.